lib/components/viewers/stop-viewer.js

```diff
@@ -71,7 +71,7 @@ class StopViewer extends Component {
 
   componentDidMount () {
```

Comment:
I think this should fire on component update as well, as there are cases where the stop viewer is reloaded but not initially rendered which causes the nearby information to disappear!